starter/src/styles/styles.pcss

```diff
@@ -1,9 +1,3 @@
 /*
 IMPORTANT: Stark styles are provided as PostCSS styles so they should be imported in a PCSS file!
 */
-
-/* Stark variables */
-@import "../../node_modules/@nationalbankbelgium/stark-ui/assets/styles/_variables.pcss";
-
-/* Stark components */
-@import "../../node_modules/@nationalbankbelgium/stark-ui/src/modules/app-logo/components/app-logo.component.pcss";
```

Comment:
I think we should keep this otherwise we'll lose the styles for app-logo component.